test/Interop/Cxx/header-defined-functions/Inputs/header-defined-functions.h

```diff
@@ -0,0 +1,15 @@
+inline int Answer() { return 42; }
```

Comment:
General notes for the tests:
1. Let's match the existing naming convention. TitleCase for classes and camelCase for functions.
2. Name the functions based on what they do. So this might be named `returns42` and the next one might be `passThroughArg`. 
3. `header-defined-functions` doesn't really explain what's being tested here. This is testing when a data member's initializer uses an inline function, so maybe this should go in `Cxx/class/initializer-from-inline.h`.

Reply:
@zoecarver I've addressed most of the review feedback. Ready for more.